src/Microsoft.AspNetCore.Blazor.Templates/Directory.Build.props

```diff
@@ -1,8 +1,7 @@
 <Project>
-  <!--
-    Left mostly blank to avoid sharing config with the build for the main repo.
-    For example, we want the templates to declare their own independent package restore sources.
-  -->
+  <PropertyGroup>
+    <TemplateBlazorPackageVersion>0.2.0-preview1-10195</TemplateBlazorPackageVersion>
```

Comment:
Consider defining this at the top level, or where our other versions are defined. Right now we have versions in a few places